css/src/tokens/font-stack.scss

```diff
@@ -3,7 +3,7 @@
  */
 $monospace-font-stack: 'SFMono-Regular', 'Consolas', 'Liberation Mono', 'Menlo', 'Courier',
 	monospace !default;
-$normal-font-stack: -apple-system, 'BlinkMacSystemFont', 'Segoe UI Variable Text', 'Segoe UI',
+$normal-font-stack: 'Segoe UI', 'Segoe UI Variable Text', -apple-system, 'BlinkMacSystemFont',
```

Comment:
Should `Segoe UI Variable Text` come before `Segoe UI` as it's the default font for Windows 11?

Reply:
good question, not all windows machines have `Segoe UI Variable Text`, only windows 11. So do we want to load an extra font file on non-windows 11 machines?

Reply:
hmm, I'm unable to get `Segoe UI Variable Text` to pull from the local machine's font, even though I'm using windows 11. For the best performance, I think all windows machines should keep using `Segoe UI` and only load in `Segoe UI Variable Text` on the machines that don't have `Segoe UI`. so I think `Segoe UI` before `Segoe UI Variable Text` would be best